clang/docs/ReleaseNotes.rst

```diff
@@ -647,7 +647,11 @@ Improvements to Clang's diagnostics
   #GH69470, #GH59391, #GH58172, #GH46215, #GH45915, #GH45891, #GH44490,
   #GH36703, #GH32903, #GH23312, #GH69874.
 
-  
+- Improved the performance of static assertions involving large integers by
+  using hex format instead of string for particularly large values.
+
```

Comment:
Don't do a newline here I think?  If you do it separates the things during rendering.